hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/S3SecretManager.java

```diff
@@ -27,4 +27,10 @@
 public interface S3SecretManager {
 
   S3SecretValue getS3Secret(String kerberosID) throws IOException;
+
```

Comment:
The two methods are confusing a little. Especially as the bigger part of the implementation is duplicated. Would be great to merge them (or use better naming). (Not a blocker, we can address it later).

Reply:
Renamed new api to getS3UserSecretString, open to any better name you may suggest. Purpose of both api's is different so consolidating them right not might not be a good option. We can discuss this further in separate jira.